composer.json

```diff
@@ -16,7 +16,9 @@
   "require-dev": {
     "doctrine/coding-standard": "^4.0",
     "phpunit/phpunit": "^7.2",
-    "psr/http-message": "^1.0"
+    "phpstan/phpstan-shim": "^0.10",
+    "psr/http-message": "^1.0",
+    "react/promise": "2.*"
```

Comment:
I like how you resolved those missing dependencies. Them being missing worried me for a long time. No more 👍